### DISCUSSION TITLE
Is it possible to connect using LDAP connection string like this?

### DISCUSSION THREAD

**Discussion by Jamtastic808** [flamencist/ldap4net#150] [Q&A]
Hello,
I am wondering if someone could kindly help me out.  I am new to Ldap4net and also LDAP/Active Directory.

I am trying to port some code.
Existing is using .NET's DirectoryEntry and I have domain paths stored in DB like this format:
LDAP://DC=|SERVER NAME|[,DC=|EXTENSION|]

e.g. 
`var directorEntry = new DirectoryEntry("LDAP://DC=XYZ,DC=net", "myUser", "myPassword");`

I have tried all different ways and I am not able to connect with this format. Any help much appreciated!

**Comment by flamencist** [comment]
Please start from documentation and the example https://github.com/flamencist/ldap4net/blob/master/Examples/C%23/Program.cs
